docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -133,3 +133,27 @@ You will need a ssh agent running with access to the corresponding private key.
 .. prompt:: bash $
 
   admin/run-acceptance-tests --distribution fedora-20 --provider aws --config-file config.yml
+
+
+DigitalOcean
+------------
+
+To run the acceptance on DigitalOcean, you need:
+
+- a DigitalOcean account,
+- a "Legacy API v1" Client ID and API key
+  (https://cloud.digitalocean.com/generate_api_key),
+- an "API v2" token, which will be used to update the kernel of new droplets,
+  (https://cloud.digitalocean.com/settings/applications), and
+- an ssh-key registered with the AWS account.
+
+.. code-block:: yaml
+
+   digitalocean:
+     client_id: "<DigitalOcean API v1 client id>"
+     api_key: "<DigitalOcean API v1 api key>"
+     token: "<DigitalOcean API v2 api token>"
+     location: "<DigitalOcean location slug e.g. lon1>"
```

Comment:
How do I best choose the location? I guess I look at a list of slugs somewhere and choose the one which seems closest to me.